timescaledb/quick-start/java.md

```diff
@@ -0,0 +1,621 @@
+        var conn = DriverManager.getConnection(connUrl);
+        System.out.println(conn.getClientInfo());
+    }
+}
+```
+
+Run with the `java -cp *.jar Main.java` command 
+and you should see this output: `{ApplicationName=PostgreSQL JDBC Driver}`.
+
+Congratulations, you've successfully connected to TimescaleDB using Java.
+
+## Create a relational table [](create_table)
+
+### Step 1: Formulate your SQL statement
+
+First, compose a string which contains the SQL state that you would use to create a relational table.
+In this example, we create a table called `sensors`, with columns `id`, `type` and `location`:
+
+```sql
+CREATE TABLE sensors (
+    id SERIAL PRIMARY KEY,
+    type TEXT NOT NULL,
+    location TEXT NOT NULL
+)
+```
+
+### Step 2: Execute the SQL statement and commit changes
+
+Next, execute the `CREATE TABLE` query by creating a statement,
+executing the query from `Step 1` and check that table was created with `SELECT` statement:
+
+```java
+package com.timescale.java;
+
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class Main {
+
+    public static void main(String... args) throws SQLException {
+        var connUrl = "jdbc:postgresql://localhost:5432/postgres?user=postgres&password=postgres";
+        var conn = DriverManager.getConnection(connUrl);
+
+        var createSensorTableQuery = """
+                CREATE TABLE sensors (
+                    id SERIAL PRIMARY KEY,
+                    type TEXT NOT NULL,
+                    location TEXT NOT NULL
+                )
+                """;
+        try (var stmt = conn.createStatement()) {
+            stmt.execute(createSensorTableQuery);
+        }
+
+        var showAllTablesQuery = "SELECT tablename FROM pg_catalog.pg_tables WHERE schemaname = 'public'";
+        try (var stmt = conn.createStatement();
+             var rs = stmt.executeQuery(showAllTablesQuery)) {
+            System.out.println("Tables in the current database: ");
+            while (rs.next()) {
+                System.out.println(rs.getString("tablename"));
+            }
+        }
+    }
+}
+```
+
+Congratulations, you've successfully created a relational table in TimescaleDB using Java.
+
+## Generate a hypertable [](generate_hypertable)
+
+In TimescaleDB, the primary point of interaction with your data is a [hypertable][timescaledb-hypertable],
+the abstraction of a single continuous table across all space and time intervals, 
+such that one can query it via standard SQL.
+
+Virtually all user interactions with TimescaleDB are with hypertables.
+Creating tables and indexes, altering tables, inserting data, and selecting data,
+can (and should) all be executed on the hypertable.
+
+A hypertable is defined by a standard schema with column names and types, 
+with at least one column specifying a time value.
+
+### Step 1: Formulate the CREATE TABLE SQL statement for your hypertable
```

Comment:
Maybe we could avoid the repetition of `Formulate ... statement for your hypertable` in all the subtitles?

I'd call like:

```md
### Step 1: Create `sensor_data` table with timestamp
...
### Step 2: Call `create_hypertable` for `sensor_data` table
...
### Step 3: Wrap previous steps into a java application
...
```

Or without emphasis in the terms:

```md
### Step 1: Create sensors data table
...
### Step 2: Create hypertable for sensors data
...
### Step 3: Execute previous steps from your Java code
```

Reply:
Sure, I totally agree. Almost all text is copy-paste from other quick-starts. I just tried to repeat everything from already exist guides but on java language :)

Reply:
Yes! they're full of issues and let's try to improve from here! Thanks again for your patience!